src/big/number.cr

```diff
@@ -4,6 +4,10 @@ struct BigInt
 end
 
 struct BigFloat
+  def fdiv(other : Number::Primitive) : self
+    self.class.new(self / other)
```

Comment:
Why the `self.class.new`?

Reply:
Because copy-past of abstract code, but I actually like it that way. Same as why annotate with `: self` in the return type.

I would leave it that way.